packages/beacon-node/src/chain/stateCache/stateContextCache.ts

```diff
@@ -21,7 +21,7 @@ export class StateContextCache {
   private readonly cache: MapTracker<string, CachedBeaconStateAllForks>;
   /** Epoch -> Set<blockRoot> */
   private readonly epochIndex = new Map<Epoch, Set<string>>();
-  private readonly metrics: Metrics["stateCache"] | null | undefined;
+  private readonly metrics: (Metrics["stateCache"] & Metrics["epochCache"]) | null | undefined;
```

Comment:
Why not add this metrics to the epochCache?

Reply:
> Why not add this metrics to the epochCache?

You mean why not add `epochCache` metric under `stateCache` metrics? 

Because `EpochCache` is quite a different thing from `StateContextCache` and they are under different packages.